lib/zwave/ZwaveDevice.js

```diff
@@ -318,7 +343,8 @@ class ZwaveDevice extends MeshDevice {
 	 * @private
 	 */
 	_parseSetting(settingObj = {}, value) {
-		let parser, customParser
+		let parser,
+			customParser;
```

Comment:
Where is `customParser` used?

Reply:
A couple of lines down, in the same `_parseSetting` method.